source/nodejs/adaptivecards-ui-testapp/index.html

```diff
@@ -6,12 +6,71 @@
 
 		<script src="https://unpkg.com/markdown-it@8.4.0/dist/markdown-it.js"></script>
```

Comment:
not part of this review -- feel free to move to a bug:

retrieving `markdown-it` over the wire might not be the best idea in CI. it's probably better to `npm install markdown-it` in this project and get it bundled with webpack

Reply:
most of the places where this library is used are not executed/run/opened during CI or any kind of build, does it make sense to open the bug for all projects

Reply:
I wouldn't worry about it, and it's not a problem that really needs to be solved right now :)